app/editor/src/components/form/options/interfaces/IOptionItem.ts

```diff
@@ -2,4 +2,5 @@ export interface IOptionItem<T extends string | number = string | number> {
   discriminator: 'IOption';
   label: string;
   value?: T;
+  isEnabled: boolean;
```

Comment:
Not fully implemented, but we need to handle disabled options.  If an option is disabled it shouldn't be displayed as an option unless the current value is the disabled one.